tachikoma-it/src/test/kotlin/com/sourceforgery/tachikoma/database/dao/EmailStatusEventDAOTest.kt

```diff
@@ -105,4 +105,24 @@ class EmailStatusEventDAOTest : DIAware {
 
         assertEquals(0, emailStatusEvents.size)
     }
+
+    @Test
+    fun `find entries based on tags`() {
+        val authentication1 = daoHelper.createAuthentication("example.org")
+        daoHelper.createEmailStatusEvent(
```

Comment:
You should create a second event which does not have tag `B` I think

Reply:
Done